src/downloadManager.ts

```diff
@@ -81,7 +81,7 @@ export class DownloadManager {
         const destPath = this.getToolPackagesPath(["dist"]);
         const absolutePath: string = this.getToolPackagesPath(["dist", fileName]);
 
-        await utils.checkFileExists(absolutePath).then(async (pkgExists) => {
+        await pathExists(absolutePath).then(async (pkgExists) => {
```

Comment:
maybe remove `await` since we are not using the result return and using `.then()` to process the result instead

Reply:
But we need for the process to wait to finish or else the install part will just execute (on Windows has happened on Mac it doesn't) before this part being finished.